rotkehlchen/tests/utils/dataimport.py

```diff
@@ -1641,6 +1705,102 @@ def assert_binance_import_results(rotki: Rotkehlchen):
             notes='Imported from binance CSV file. Binance operation: POS savings interest',
         ), HistoryEvent(
             identifier=6,
+            event_identifier='BNC_ac8a1a8ccd32547766f312428a59ac8ea667f72c33c0c4695b3fc82a7896645f',
+            sequence_index=0,
+            timestamp=ts_sec_to_ms(Timestamp(1640910825)),
+            location=Location.BINANCE,
+            asset=A_USDT,
+            balance=Balance(amount=FVal(500.00000000)),
+            event_type=HistoryEventType.TRANSFER,
+            event_subtype=HistoryEventSubType.NONE,
+            location_label='CSV import',
+            notes='Transfer Between Main and Funding Wallet',
+        ), HistoryEvent(
```

Comment:
What is the purpose of these events?

`BNC_ac8a1a8ccd32547766f312428a59ac8ea667f72c33c0c4695b3fc82a7896645f`?

It's a "transfer" and appears twice. So this is wrong.

If it's internal transfers we:

1. Probably don't need to show them. We don't care about them.
2. We definitely should not show them duplicated.